Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -8,10 +8,117 @@
 Shutdown
 #######################
 
```

Comment:
May be a quick overview of the shutdown and park of auxtel means would be helpful and clarifying? What do you think?

Reply:
I agree with that. The intro should include what the shutdown does and where the park position is. In the introduction it should be clear what the expected outcome is.